app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -19,18 +19,31 @@
             app:title="@string/subscriptions_label"
             app:navigationIcon="?homeAsUpIndicator" />
 
-        <TextView
+        <LinearLayout
             android:id="@+id/feeds_filtered_message"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:paddingBottom="8dp"
-            android:text="@string/filtered_label"
-            android:layout_marginTop="-12dp"
             android:layout_marginLeft="60dp"
             android:layout_marginStart="60dp"
+            android:layout_marginTop="-12dp"
             android:background="?android:attr/selectableItemBackground"
-            android:visibility="gone"
-            tools:visibility="visible" />
+            android:importantForAccessibility="no"
```

Comment:
I assume this was a suggestion from Android Studio? I would move this to the ImageView. Currently, this makes the entire message inaccessible with TalkBack, I think

Reply:
Yeah, although it was when I added the LinearLayout to `nav_section_layout.xml` - I just copied it across here. Would you like me to remove it from both?

Reply:
Yeah, removing would be good. I overlooked it over in the other PR.

Reply:
Cool. Will remove from both in this PR.